velox/substrait/tests/VeloxSubstraitRoundTripTest.cpp

```diff
@@ -86,6 +110,33 @@ TEST_F(VeloxSubstraitRoundTripTest, project) {
   assertPlanConversion(plan, "SELECT c0 + c1, c1 / c2 FROM tmp");
 }
 
+TEST_F(VeloxSubstraitRoundTripTest, cast) {
+  auto vectors = makeVectors(3, 4, 2);
+  createDuckDbTable(vectors);
+  // Cast int32 to int64.
+  auto plan1 =
```

Comment:
Naming variables a1, a2, a3 is not the best practice. Let's define a single 'plan' variable and re-use it.